src/components/wallet/Delegate.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-show="Object.keys(delegate).length">
+  <div v-show="delegate">
```

Comment:
```suggestion
  <div v-if="delegate">
```

to avoid `TypeError`s (`v-show` renders always, but it is hidden).